components/tools/OmeroPy/test/integration/clitest/test_tag.py

```diff
@@ -41,11 +44,16 @@ def teardown_method(self, method):
         self.teardown_mock()
 
     @classmethod
+    def new_tag(self, name=""):
```

Comment:
Yes, I did find a similar method in OmeroWeb tests but that works very differently. I guess it should stay here until it is used somewhere else.